orchagent/crmorch.cpp

```diff
@@ -64,7 +64,7 @@ const map<CrmResourceType, uint32_t> crmResSaiAvailAttrMap =
     { CrmResourceType::CRM_SNAT_ENTRY, SAI_SWITCH_ATTR_AVAILABLE_SNAT_ENTRY },
     { CrmResourceType::CRM_DNAT_ENTRY, SAI_SWITCH_ATTR_AVAILABLE_DNAT_ENTRY },
     { CrmResourceType::CRM_MPLS_INSEG, SAI_OBJECT_TYPE_INSEG_ENTRY },
```

Comment:
Is the INSEG_ENTRY correct?

Reply:
yes, label routes usually have a separate tables than IP routes.

Reply:
but in code it doesn't look right, will check further.

Reply:
There is no SAI attribute for AVAILABLE INSEG entries. How are such cases handled. Should the code for inseg label CRM be removed?